ui/src/components/MainContainer.js

```diff
@@ -39,9 +39,13 @@ import AdminTestimonyContainer from './admin/testimony/AdminTestimonyContainer';
 import TextDetails from './witness/text-testimony/TextDetails';
 import AdminAnnouncementContainer from './admin/announcements/AdminAnnouncementContainer';
 import PrayerFloatButton from './connect-prayer/PrayerFLoatButton';
-import RippleOutContainer from './ripple-out/RippleOutContainer';
 import AdminFundraiseContainer from './admin/fundraise/AdminFundraiseContainer';
 import AdminFaqContainer from './admin/faqs/AdminFaqContainer';
+import RippleOutStorySection from './ripple-out/RippleOutStoryContainer';
+import RippleOutStepsSection from './ripple-out/RippleOutStepsContainer';
+import RippleOutSpaceSection from './ripple-out/RippleOutSpaceContainer';
```

Comment:
how come this one is still SpaceSection? but you deleted it and changed it to Container

Reply:
lol how is it working

Reply:
LOL idk...